jerry-core/parser/regexp/re-parser.cpp

```diff
@@ -403,11 +467,12 @@ re_parse_char_class (re_parser_ctx_t *parser_ctx_p, /**< number of classes */
         append_char_class (re_ctx_p, 0x2030UL, 0x205EUL);
         append_char_class (re_ctx_p, 0x2060UL, 0x2FFFUL);
         append_char_class (re_ctx_p, 0x3001UL, 0xFEFEUL);
```

Comment:
@LaszloLango, I mean, we should not put actual range values to `re_ctx_p`, instead we could just put identifier of range, like `'S'`, and in `re_match_regexp` we then could use the `lit_char_is_white_space` and `lit_char_is_space_separator`-like interfaces.

I think, it is better to define missing interfaces in lit-char-helpers.h. Could you, please, clarify list of missing interfaces / ranges?